src/Perspectives/immutable/ungoverned/cluster/ClusterConservativeWithRecognizedCollateralsPerspective.sol

```diff
@@ -0,0 +1,145 @@
+        adapterRegistry = AdapterRegistry(adapterRegistry_);
+        irmFactory = EulerKinkIRMFactory(irmFactory_);
+        recognizedCollateralPerspectives = recognizedCollateralPerspectives_;
+    }
+
+    function name() public pure virtual override returns (string memory) {
+        return "Immutable.Ungoverned.ClusterConservativeWithRecognizedCollateralsPerspective";
+    }
+
+    function perspectiveVerifyInternal(address vault) internal override {
+        // the vault must be deployed by recognized factory
+        testProperty(vaultFactory.isProxy(vault), ERROR__FACTORY);
+
+        // verify vault configuration at the factory level
+        GenericFactory.ProxyConfig memory config = vaultFactory.getProxyConfig(vault);
+
+        // cluster vaults must not be upgradeable
+        testProperty(!config.upgradeable, ERROR__UPGRADABILITY);
+
+        // verify vault configuration at the governance level
+        // cluster vaults must not have a governor admin
+        testProperty(IEVault(vault).governorAdmin() == address(0), ERROR__GOVERNOR);
+
+        // cluster vaults must have an interest fee in a certain range. lower bound is enforced by the vault itself
+        testProperty(IEVault(vault).interestFee() <= 0.5e4, ERROR__INTEREST_FEE);
+
+        // cluster vaults must point to a Kink IRM instance deployed by the factory
+        testProperty(irmFactory.isValidDeployment(IEVault(vault).interestRateModel()), ERROR__INTEREST_RATE_MODEL);
+
+        {
+            // cluster vaults must not have supply or borrow caps
+            (uint32 supplyCap, uint32 borrowCap) = IEVault(vault).caps();
+            testProperty(supplyCap == 0, ERROR__SUPPLY_CAP);
+            testProperty(borrowCap == 0, ERROR__BORROW_CAP);
+
+            // cluster vaults must not have a hook target nor any operations disabled
+            (address hookTarget, uint32 hookedOps) = IEVault(vault).hookConfig();
+            testProperty(hookTarget == address(0), ERROR__HOOK_TARGET);
+            testProperty(hookedOps == 0, ERROR__HOOKED_OPS);
+        }
+
+        // cluster vaults must not have any config flags set
+        testProperty(IEVault(vault).configFlags() == 0, ERROR__CONFIG_FLAGS);
+
+        // cluster vaults must have certain liquidation cool off time
+        testProperty(IEVault(vault).liquidationCoolOffTime() == 1, ERROR__LIQUIDATION_COOL_OFF_TIME);
+
+        // cluster vaults must point to an ungoverned EulerRouter instance deployed by the factory
+        address oracle = IEVault(vault).oracle();
+        testProperty(routerFactory.isValidDeployment(oracle), ERROR__ORACLE);
+        testProperty(IEulerRouter(oracle).governor() == address(0), ERROR__ORACLE);
+
+        // Verify the unit of account is either USD or WETH
+        address unitOfAccount = IEVault(vault).unitOfAccount();
+        testProperty(unitOfAccount == USD || unitOfAccount == ETH || unitOfAccount == WETH, ERROR__UNIT_OF_ACCOUNT);
```

Comment:
Let's just have `USD` and `WETH`. 

`ETH` and `WETH` are interchangeable, DeFi users and frontends accept them as such. We don't want to run into a situation where a router wants to price WETH/ETH.

Reply:
Good point. Why WETH and not ETH?

Reply:
Oh I guess there is no standardised ETH address, unless we want to use the `0xeeeee...` one.

Reply:
yes, and it's pretty uncommon that defi protocols use the 0xeeeee address to represent eth

Reply:
they just use WETH